cordapp-example/workflows-kotlin/src/main/kotlin/com/example/flow/CreateMoneyFlow.kt

```diff
@@ -29,7 +29,7 @@ object CreateMoneyFlow {
 
             txBuilder.verify(serviceHub)
             val signedTx = serviceHub.signInitialTransaction(txBuilder)
-            val setOfSessions: Set<FlowSession> = cashOwners.map { it.key }.map { initiateFlow(it) }.toSet()
+            val setOfSessions = cashOwners.map { it.key }.map { initiateFlow(it) }
```

Comment:
Instead of double `.map`: `.map { initiateFlow(it.key) }`

Reply:
first map is for removing duplicates, so I have only one flow per key

Reply:
That would be true if there was a `.distinct` but there is none.